tests/integration/test_parametrize.tavern.yaml

```diff
@@ -219,3 +219,31 @@ stages:
       status_code: 200
       body:
         value: "{to_send}"
+
+
+---
```

Comment:
I think this could just do with 1 extra test to make sure that it can handle both `key` and `keys` in one test - eg:

```yaml
marks:
  - parametrize:
      key: abc
      vals:
        - one
        - two
  - parametrize:
      keys:
        - def
        - ghi
      vals:
        - def: three
          ghi: four
        - def: five
          ghi: six
```